google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/internal/DummyBatchingDescriptor.java

```diff
@@ -27,10 +27,7 @@
  * com.google.cloud.bigtable.data.v2.stub.EnhancedBigtableStub#mutateRowsCallable()}. The settings
  * are exposed to the user using the {@link com.google.cloud.bigtable.data.v2.models.RowMutation}
  * wrapper, but the actual descriptor works on the underlying {@link
- * com.google.bigtable.v2.MutateRowsRequest}s. This class is used as a placeholder for the settings
- * and is replaced with the actual implementation of {@link
- * com.google.cloud.bigtable.data.v2.stub.bulkmutaterows.BulkMutateRowsBatchingDescriptor} when
```

Comment:
We can just remove the `@link` here - I think the linked class might exist in a later PR by @igorbernstein2 .

Reply:
Oh good to know.
Just removed the `@link`.

Reply:
I don't see the class reference still...

Reply:
Originally it was a dangling link in the bulk mutations surface PR, meant to give context until the impl PR (#2987) was merged. During the reviews, its package was renamed, but the link was never updated.
It is fixed in PR #2987
https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2987/files#diff-e26d48d3afb81b5ddff92add0217ae58

Reply:
Ok, this time I actually pushed the commit to remove the `@link`.